.github/workflows/publish-docs.yml

```diff
@@ -40,6 +40,8 @@ jobs:
           sbt -Dakka.genjavadoc.enabled=true "+~ 2.13 publishRsync"
         env:
           SCP_SECRET: ${{ secrets.SCP_SECRET }}
+          PUBLISH_USER: ${{ secrets.PUBLISH_USER }}
+          PUBLISH_PASSWORD: ${{ secrets.PUBLISH_PASSWORD }}
```

Comment:
I'm not sure why those are needed for docs, but they are https://github.com/akka/akka/actions/runs/6419707045/job/17430125255

Reply:
The setup logic runs even for docs.